src/HomePagePanel.js

```diff
@@ -1,81 +1,73 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import CardContainer from './CardContainer'
+import PanelContent from './PanelContent'
 import PanelBar from './PanelBar'
 
-import AtlasHPCard from 'atlas-homepage-cards'
-
 class HomePagePanel extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-    	panelNameA: `Species`,
-    	panelNameB: `Featured`
+    	speciesTabName: Object.keys(this.props.speciesResources)[0],
+    	experimentTabName: Object.keys(this.props.experimentResources)[0]
     }
     this.changePanel = this.changePanel.bind(this)
   }
 
   changePanel(event){
-  	this.props.panelNameAList.includes(event.target.id ) ?
-  	this.setState({
-  		panelNameA:  event.target.id
-  	}) :
-  	this.setState({
-  		panelNameB:  event.target.id
-  	})
+  	this.props.speciesResources[event.target.id] ?
+    	this.setState({speciesTabName:  event.target.id}) :
+    	this.setState({experimentTabName:  event.target.id})
   }
 
   render() {
-  	var {panelNameAList, panelNameBList, host} = this.props
+  	const {host, speciesResources, experimentResources, SpeciesCardType, LatestExperimentCardType} = this.props
   	return (
-  		<div className="row expanded margin-top-large" data-equalizer>
-
-	    	 <div className="small-12 medium-12 large-6 columns">
-				<div className="callout experiment-list-latest padding-bottom-for-button" data-equalizer-watch>
-
-					<PanelBar key="species-bar" panelNames={panelNameAList} onClick={this.changePanel}/>
-				    
-				    <div className="tabs-content" data-tabs-content="browse-by-tabs">
-				        <CardContainer key="species" panelName={this.state.panelNameA} CardType={AtlasHPCard} host={host}/>
-				    </div>
-			
-			       	<div className="row align-row-to-bottom">
-			         <div className="small-6 small-centered columns margin-top-large">
-			            <a id="viewAllLink" href={`${host}experiments`} className="button float-center">View all species</a>
-			         </div>
-			        </div>
+  		<div className={`row expanded margin-top-large`} data-equalizer>
```

Comment:
Since we’re using now rows the `<div>` structure can be simplified considerably. Just use rows and the panels inside, that’s it. No need for callouts or `data-equalizer`.